src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -547,8 +547,8 @@ public void injectMediaToBrowserstackDevice(String uploadFileURL) {
         }
     }
 
-    public void scrollInDynamicLayer(Direction direction) {
-        Dimension dimension = driver.manage().window().getSize();
+    public void scrollInDynamicLayer(Direction direction, WebElement dropDownElement) {
```

Comment:
why is the element name dropDownElement?

Reply:
mostly the dynamic layer is an inner dropdown, should I change the name to dynamicLayerElement?